docs/source/stats-collection.rst

```diff
@@ -0,0 +1,72 @@
+.. _stats_collection:
+
+Comparing Spider Executions
+===========================
+
+Sometimes it is worthy to compare results from previous executions of your
+spider. For example, we should be able to track whether the number of items
+returned is lower than previous executions, which may indicate a problem.
+
+Scrapy allows you to collect stats in the form key/value, but these stats are
+lost when the spider finishes its execution. Spidermon provides a built-in
+stats collector that stores the stats of all spider executions in your local
+file system.
+
+After enabling this stats collector, every spider instance running will have a
+``stats_history`` attribute containing a list of the latest spider executions
+that can be easily accessed in your monitors.
+
+To enable it, include the following code in your project settings:
+
+.. code-block:: python
+
+    # myproject/settings.py
+    STATS_CLASS = (
+        "spidermon.contrib.stats.statscollectors.LocalStorageStatsHistoryCollector"
+    )
+
+    # Stores the stats of the last 10 spider execution (default=100)
+    SPIDERMON_MAX_STORED_STATS = 10
+
+The following example shows how we can verify whether the number of items
+returned in the actual spider execution is higher than 90% of the mean of items
+returned in the latest spider executions.
+
+.. code-block:: python
+
+    # myproject/monitors.py
+
+    @monitors.name("History Validation")
+    class HistoryMonitor(Monitor):
+
+      @monitors.name("Expected number of items extracted")
+      def test_expected_number_of_items_extracted(self):
```

Comment:
can we now make this monitor part of base monitors ? We have already have a monitor to test the given number of items but it would be nice addition to check coverage based on historical crawls.

Reply:
Separate issue could be created for this.

Reply:
NVM, i see #122